index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <figure id="img-div">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
```

Comment:
Please, kindly consider adding an alternative text on your `<img>` tag because The `<img>` alt attribute is used to specify the alternate text for an image. It is useful when the image is not displayed. It is used to give alternative information for an image.